login_server/loginserver.py

```diff
@@ -330,6 +330,22 @@ def handle_http_request_message(self, msg):
                 'online_players': len(self.players),
                 'online_servers': len(self.game_servers)
             }, sort_keys=True, indent=4))
+        elif msg.env['PATH_INFO'] == '/detailed_status':
+            online_game_servers_list = [
+                {'locked':      gs.password_hash is not None,
+                 'mode':        gs.game_setting_mode,
+                 'name':        gs.description,
+                 'map_id':      gs.map_id,
+                 'players':     [p.display_name for p in gs.players.values()],
+                 'nplayers':    len(gs.players)} for gs in self.game_servers.values()
```

Comment:
Why send the length when the receiver can just do len(players)?

Reply:
The same for online_players and online_servers.

Reply:
Good point, the nplayers and online_servers keys can be omitted from the output. 
The online_players key is useful because online_players - len(players) = number of players in the lobby.

Reply:
Did you mean "online_players"? I'm saying that all the things that use "len" are not needed, because the client can just do len himself.

Reply:
Yep, you're right. I've removed all the keys that just use len( ) from the "/detailed_status" response. I think we might need to leave the "/status" response section alone though, since there are a few existing services that (I think) use it just to show a player count number.

Reply:
Yeah that's fine. I prefer that.